test/expect/http/dune

```diff
@@ -0,0 +1,11 @@
+(library
+ (name test_expect_http)
+ (libraries
+  base
+  dream
+  dream.http
+  lwt
+  lwt.unix
+  ppx_expect.common
+  ppx_inline_test.config
+  ppx_expect.config_types))
```

Comment:
leaving these dune files here for the future instead of having to set them up again.